src/routes/console/project-[project]/overview/+layout.svelte

```diff
@@ -89,14 +89,14 @@
 
                                 <div class="grid-item-1-end-start">
                                     <div class="heading-level-4">
-                                        {format(last($usage.documents).value)}
+                                        {format(total($usage.documents))}
```

Comment:
the one for documents should be fine 👍🏻 they dont need any change

Reply:
I think only executions are different

Reply:
All these should be summed, as all these are now using time-series data, before we got sum from database and data already had the sum, now it's not

Reply:
I believe that after usage refactor the is no more aggregation and we ONLY have time series data. So we should be summing everything.

Reply:
Of course, not the best solution since we don't have real total, just total in time period. But its a patch until usage container can properly sum.